web-frontend/src/pages/TransferCoins/SendCoins/SendRedirect.tsx

```diff
@@ -0,0 +1,51 @@
+import { useNavigate } from 'react-router-dom';
+import { useResource } from '../../../custom/api';
+import { AccountObject } from '../../../models/AccountModel';
+import { routeFor } from '../../../utils';
+import { useState } from 'react';
+import Intl from '../../../i18n/i18n';
+
+function Redirect() {
+  const navigate = useNavigate();
+  const [nickname, setNickname] = useState<string>('');
+  let error: string | null = null;
+  const { data: accounts = [] } = useResource<AccountObject[]>('accounts');
+  const queryParams = new URLSearchParams(location.search);
+  const sender = queryParams.get('sender');
+  if (sender) {
```

Comment:
Hey @Thykof need you to check here because I have not to much context fo that in the BE side.

Reply:
nothing special